README.md

```diff
@@ -90,25 +90,11 @@ When the user is browsing another site (publisher.com), that site (or issuer.com
 document.hasTrustToken(<issuer>)
 ```
 
-
 This returns whether there are any valid trust tokens for a particular issuer, so that the publisher can decide whether to attempt a token redemption.
 
+#### Redeeming tokens from the issuer, obtaining a signed attestation
 
-```
```

Comment:
This is pretty core to the PrivacyPass protocol flow, and should probably remain as part of the core explainer rather than an extension.

Reply:
OK; I'll make it more explicit in the design doc which bits of the explainer we're implementing and which we aren't.

Reply:
Reverted this pat of the change.